.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: generic
 
 install:
-  - ci/install.sh
+  - scripts/install.sh
```

Comment:
Can't say that I like it, but ...

Reply:
Any suggestions ? My thinking was that these are not really `ci` specific anymore, as you can run them locally etc. and looking at a sample balena project (rendition), they use `scripts` as well for this.

Reply:
Nah, I'm fine with it, just a nitpick.